tests/components/utility_meter/test_sensor.py

```diff
@@ -638,82 +638,89 @@ async def test_restore_state(
     # Home assistant is not runnit yet
     hass.set_state(CoreState.not_running)
 
-    last_reset = "2020-12-21T00:00:00.013073+00:00"
+    last_reset_1 = "2020-12-21T00:00:00.013073+00:00"
+    last_reset_2 = "2020-12-22T00:00:00.013073+00:00"
 
     mock_restore_cache_with_extra_data(
         hass,
         [
+            # sensor.energy_bill_tariff1 is restored as expected
             (
                 State(
-                    "sensor.energy_bill_onpeak",
-                    "3",
+                    "sensor.energy_bill_tariff1",
+                    "1.1",
                     attributes={
                         ATTR_STATUS: PAUSED,
-                        ATTR_LAST_RESET: last_reset,
-                        ATTR_UNIT_OF_MEASUREMENT: UnitOfEnergy.KILO_WATT_HOUR,
+                        ATTR_LAST_RESET: last_reset_1,
+                        ATTR_UNIT_OF_MEASUREMENT: UnitOfEnergy.MEGA_WATT_HOUR,
                     },
                 ),
                 {
                     "native_value": {
                         "__type": "<class 'decimal.Decimal'>",
-                        "decimal_str": "3",
+                        "decimal_str": "1.2",
```

Comment:
Use different value so we know if the value is restored from the `UtilitySensorExtraStoredData`